website/docs/tutorials/quickstart.md

```diff
@@ -4,19 +4,25 @@ title: Quickstart
 
 In this section we will attempt to guide you in order to get started with Unleash easily. There are multiple options to get started with Unleash, browse the headings to find the method that works best for you.
 
-## I just want to get started creating toggles without much setup
+:::info Feature Flags or Feature Toggles?
+
+This document uses Feature Flags and Feature Toggles interchangeably. Some people prefer flag; others prefer toggle. We use both - they are synonyms for us.
```

Comment:
As a discussion topic: why have you capitalized "Feature Flag" and "Feature Toggle" here? They're normal nouns, so I don't see why they should be capitalized. I think we could call attention to them as terms either by **bolding them** or by "surrounding them with quotes".

However, I'm very open to hearing if you have a different perspective on this.

I'd suggest something like this: 

```suggestion
:::info Feature flags or feature toggles?

This document uses **feature flags** and **feature toggles** interchangeably. Some people prefer flag; others prefer toggle. We use both - they are synonyms for us.
```